### DISCUSSION TITLE
nm_usb_get_serial: Access denied (insufficient permissions)

### DISCUSSION THREAD

**Discussion by mckaygerhard** [nemuTUI/nemu#175] [Q&A]
I tyry to add a usb drive attached.. but seems the wiki si not so clear about https://github.com/nemuTUI/nemu/wiki/Running-as-non-root

the group USB seems is not always created seems

**Comment by 0x501D** [comment]
The instructions assume that the system has a 'usb' group; if this is not the case, then it must be additionally created manually.

**Reply by mckaygerhard** [reply]
umm i search for a while and usb group is not so standard .. i think i need to do some investigations and perform a better wiki.. i will do it after some test in my side.. but the current wiki is not so selft explanatory
